src/main/scala/repro/overloadedmap.scala

```diff
@@ -23,25 +23,21 @@ object Expr2:
 
   case class AggProject[A <: AnyNamedTuple]($a: A) extends Expr2[NamedTuple.Map[A, StripExpr2], NExpr]
   case class Project[A <: AnyNamedTuple]($a: A) extends Expr2[NamedTuple.Map[A, StripExpr2], NExpr]
-  case class Ref[A, S<: ExprShape]() extends Expr2[A, S]
-
+  case class Ref[A, S <: ExprShape]() extends Expr2[A, S]
 
   type StripExpr2[E] = E match
     case Expr2[b, s] => b
 
   type IsTupleOfExpr[A <: AnyNamedTuple] = Tuple.Union[NamedTuple.DropNames[A]] <:< Expr2[?, NExpr]
-  extension [A <: AnyNamedTuple : IsTupleOfExpr](x: A)
-    def toRow: Project[A] = ???
+  extension [A <: AnyNamedTuple: IsTupleOfExpr](x: A) def toRow: Project[A] = ???
 
   type IsTupleOfAgg[A <: AnyNamedTuple] = Tuple.Union[NamedTuple.DropNames[A]] <:< Expr2[?, ScalarExpr]
-  extension [A <: AnyNamedTuple : IsTupleOfAgg](x: A)
-    def toRow: AggProject[A] = ???
+  extension [A <: AnyNamedTuple: IsTupleOfAgg](x: A) def toRow: AggProject[A] = ???
```

Comment:
I dislike it pushing extensions into a single line

Reply:
I agree, can we turn that rule off?

Reply:
Yes, it's fixed now. But now it does
```
case class SelectQuery
  (
      project: QueryIRNode,
      from: Seq[RelationOp],
      where: Seq[QueryIRNode],
      overrideAlias: Option[String],
      ast: DatabaseAST[?]
  ) extends RelationOp:
```
instead of
```
case class SelectQuery(project: QueryIRNode,
                       from: Seq[RelationOp],
                       where: Seq[QueryIRNode],
                       overrideAlias: Option[String],
                       ast: DatabaseAST[?]) extends RelationOp:
```

I think these behaviors are governed by the same setting. I think that this is mostly acceptable.